doc/Settings.md

```diff
@@ -8,17 +8,17 @@ On the front page of the admin panel you will see a summary of information about
 These information include the amount of messages currently being processed in the queues.
 The first number is the number of messages which could not been delivered for various reasons.
 They will be resend later.
-You can have a quick glance into that second queus in the "Inspect Queue" section of the admin panel.
+You can have a quick glance into that second queues in the "Inspect Queue" section of the admin panel.
```

Comment:
Could you please change "second" into "third" as @annando had forgotten to update the docs when he introduced the new 2nd queue. It would also be nice, if you could add a sentence about the new 2nd number.

Reply:
It would be nice if I knew exactly what it was. I guess only @annando can write this part.

Reply:
He will never do, as he is one of those who say, that the code is the documentation. Whenever I ask him about documentation a giant squirrel arrives.

Reply:
Squirrel, where? :grimacing: 
The queues are:
1. Message queue of posts that couldn't be delivered by now.
2. Deferred workerqueue tasks (Is planned as a replacement for the message queue, currently only is active for AP delivery)
3. Outstanding workerqueue tasks, not containing stuff from 2.